Tasks/VsTest/helpers.ts

```diff
@@ -22,6 +22,7 @@ export class Constants {
 export class Helper {
     public static addToProcessEnvVars(envVars: { [key: string]: string; }, name: string, value: string) {
         if (!this.isNullEmptyOrUndefined(value)) {
+            tl.debug('Setting the process env var :' + name + ' to :' + value);
```

Comment:
>debug [](start = 15, length = 5)

any potential of a secret getting logged ? (access token ?)

Reply:
+1

---
In reply to: [147331143](https://github.com/Microsoft/vsts-tasks/pull/5716#discussion_r147331143) [](ancestors = 147331143)